pnpm-workspace.yaml

```diff
@@ -47,6 +47,7 @@ patchedDependencies:
 
 onlyBuiltDependencies:
   - dprint
+  - esbuild
```

Comment:
how is this involved?

Reply:
I received a warning while running `pnpm install`, so I updated the configuration to suppress it.

```
╭ Warning ─────────────────────────────────────────────────────────────────────╮
│                                                                              │
│   Ignored build scripts: esbuild.                                            │
│   Run "pnpm approve-builds" to pick which dependencies should be allowed     │
│   to run scripts.                                                            │
│                                                                              │
╰──────────────────────────────────────────────────────────────────────────────╯
```